src/lib.rs

```diff
@@ -530,6 +530,8 @@ pub mod pallet {
 		VoteNotTallied,
 		/// Time for voting has not elapsed.
 		VotingPeriodActive,
+		/// Balance must be greater than transferring amount
+		InsufficientBalance,
```

Comment:
Do we need this? I would assume that the RuntimeError type from the Token:transfer() call would be bubble up? Perhaps the return type of the functions in impls.rs can be changed wherever used to negate the need for it.

Reply:
I guess, yes we need, as From<pallet_balances::Error<T>>` is not implemented for `pallet::Error<T>`.

Reply:
https://github.com/tellor-io/tellor-pallet/pull/33 provides a suggested solution, if you are happy to merge into your PR.